src/components/MaModal/MaModal.scss

```
@@ -2,41 +2,37 @@
   position: fixed;
   top: 0;
   left: 0;
-  right: 0;
-  bottom: 0;
-
-  background-color: rgba(0, 0, 0, 0.25);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  transition: opacity 0.4s ease;
+  background-color: get-alpha-color(black, 0.5);
+  width: 100%;
+  height: 100%;
 }
 
 .modal {
-  position: fixed;
-
-  // centering
-  left: 50%;
-  top: 50%;
-  transform: translate(-50%, -50%);
+  transition: transform 0.4s ease;
+  border-radius: 0.25rem;
+  box-shadow: 0 rem(2px) rem(8px) get-alpha-color(black, 0.67);
+  background-color: get-color(white);
 
   // so that it doesn't reach end of viewport
   width: 96vw;
-
-  // temporary, stylistic stuff
-  background-color: white;
-  box-shadow: 0 25px 50px -12px rgba(0, 0, 0, 0.25);
-  border-radius: 0.5rem;
+  word-break: break-word;
 }
 
 .modal-header {
   display: flex;
   justify-content: space-between;
   align-items: flex-start;
-  padding: get-space('xxsmall');
+  padding: get-space('xsmall');
+  border-radius: 0.25rem 0.25rem 0 0;
 }
 
 .modal-header--gradient {
   background: linear-gradient(to right, get-color(orange), get-color(pink));
-  color: get-color(white);
-  border-top-left-radius: 0.5rem;
-  border-top-right-radius: 0.5rem;
+  color: white;
```

Not that it matter much, but just for the sake of consistency:

```suggestion
  color: get-color(white);
```